data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -18,3 +18,24 @@ This script will update the sources to create ``cc-oci-runtime`` packages.
 
   * cc-oci-runtime.dsc
   * cc-oci-runtime.spec
+
+OBS
```

Comment:
Please do not use the abbreviation here. It can be used on the rest of the text then.